src/LendingPool.sol

```diff
@@ -349,12 +350,12 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken, InterestRateModule {
 
         unchecked {
             //gas: can't overflow for reasonable interest rates
-            base = 1e18 + interestRate;
+            base = 1e18 + interestRatePerYear;
```

Comment:
Why add PerYear in all variable names?
Probably its cleaner to add it as a comment where we declare the variables and keep var names as short as possible

Reply:
Variable name length doesn't affect the gas cost, so why not already make the variable names explicit. It is good for our usage that we do not go back to comment where its explained. Its good for user they exactly know what value it is without looking into documentation. And most of the protocols used explicit naming in interestrate related variables.

Reply:
Longer variable names do not make code more readable in my opinion, even opposite. Let's ask the rest of the team

Reply:
@simluyt @j-vp

Reply:
as long as we're consistent it doesn't really matter for me. I think `interestRate` should be clear enough but if other protocols use peryear, I don't see a reason to deviate

Reply:
It is not to make it longer. It is to make it more explicit. Naming of variable should:
- To reduce the effort needed to read and understand source code.
- To enable code reviews to focus on issues more important than syntax and naming standards.
- To enable code quality review tools to focus their reporting mainly on significant issues other than syntax and style preferences.
It was obvious to me during yesterday's meeting. I needed to switch files to understand what interestRate means. `interestRatePerYear` can be reverted `interestRate` however, i strongly object to the rest that ratePerYear, lowSlopePerYear namings should stay.